ghost/core/test/unit/server/lib/request-external.test.js

```diff
@@ -279,10 +279,13 @@ describe('External Request', function () {
                 throw new Error('Request should have rejected with invalid url message');
             }, (err) => {
                 should.exist(err);
-                err.message.should.be.equal('URL empty or invalid.');
+                // got v11+ throws an error instead of the external requests lib
+                err.message.should.be.equal('No URL protocol specified');
             });
         });
 
+        // with got v11, for some reason this went from a GotError HTTPError with rich details to a shitty HTTPError with no data or even code
+        // TODO: maybe try a higher version? or see if this happened with v10
```

Comment:
```suggestion
```

Reply:
😅 I'll just remove the comment altogether. Your suggestion about changing to use the response error code works how we need.